padle/padle.py

```diff
@@ -0,0 +1,489 @@
+            guild_confirmation = await get_user_confirmation(ctx, "It looks like you're in a private server. "
+                                                                  "Would you like to play PADle in this server"
+                                                                  " rather than in DMs?", timeout=30,
+                                                             force_delete=False, show_feedback=True)
+            if guild_confirmation is None:
+                return await send_cancellation_message(ctx, "Confirmation timeout.")
+            if await self.config.guild(ctx.guild).allow():
+                return await ctx.send("You have already enabled PADle in this server!")
+            if not guild_confirmation:
+                await send_confirmation_message(ctx, "PADles will continue being played in DMs.")
+            else:
+                await send_confirmation_message(ctx, "PADles can now be played in this server.")
+                await self.config.guild(ctx.guild).allow.set(True)
+        confirmation = await get_user_confirmation(ctx, "Start today's (#{}) PADle game?".format(
+            await self.config.num_days()), timeout=30, force_delete=False, show_feedback=True)
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout.")
+        if await self.config.user(ctx.author).start():
+            return await send_cancellation_message(ctx, "You cannot start multiple games!")
+        if not confirmation:
+            return
+
+        prefix = ctx.prefix
+        em = discord.Embed(title="PADle #{}".format(await self.config.num_days()), type="rich",
+                           description="Guess a card with `{}padle guess <card>`!\nIf "
+                                       "you give up, use `{}padle giveup`.".format(prefix, prefix))
+        if await self.can_play_in_guild(ctx):
+            message = await ctx.send(embed=em)
+            await self.config.user(ctx.author).start.set(True)
+            await self.config.user(ctx.author).edit_id.set(message.id)
+            await self.config.user(ctx.author).channel_id.set(message.channel.id)
+            return
+        # else
+        try:
+            message = await ctx.author.send(embed=em)
+        except discord.Forbidden:
+            return await send_cancellation_message(ctx,
+                                                   "Looks like I can't DM you. Try checking your Privacy Settings.")
+        if ctx.guild is not None:
+            await send_confirmation_message(ctx, "Check your DMs!")
+        await self.config.user(ctx.author).start.set(True)
+        await self.config.user(ctx.author).edit_id.set(message.id)
+        await self.config.user(ctx.author).channel_id.set(message.channel.id)
+
+    @padle.command(aliases=["stats"])
+    async def globalstats(self, ctx, day: int = 0):
+        """Gives global stats for the PADle (optionally on a specified day)"""
```

Comment:
this would be cool to be another closable embed menu2 but if you're traumatized and don't want to it's ok